docs: add debug unit test command into package.json (#501)
paritytech/parity-signer: README.md

}
```

#### Unit Test

Run `yarn test` for all the units test.

If debugging is needed:

> 1. Insert a line of `debugger;` into the code base where might be failed.

Reviewer: ```suggestion
1. Insert `debugger;` in the code where you think it fails.
```

2. Run `yarn test:debug`

> 3. Open a new tab in Chrome with address `chrome://inspect`

Reviewer: ```suggestion
3. Open a new tab in Chrome and go to `chrome://inspect`
```

4. Click the `inspect` button of target under `Remote Target`

> 5. Back to the terminal and choose one of the node watch commands for running tests again.

Reviewer: ```suggestion
5. Back to the terminal, choose one of the node watch commands to run the tests again.
```
